### PR TITLE
Address more MP v2 deprecations

### PR DESCRIPTION
**Pull request by damassi**
Addresses https://artsyproduct.atlassian.net/browse/PLATFORM-1641

Fixes a few more MP deprecations:
- `price` -> `listPrice`. List price can return `priceCents`, `minPriceCents` or `maxPriceCents`
- `short_description` -> `shortDescription`
- `to_s` -> `formattedMetadata`

The last remaining deprecations (to be fixed in https://github.com/artsy/reaction/pull/2552):

Screen Shot 2019-07-31 at 12 08 10 PM

### CODE REVIEW

File: src/Apps/Collect/Components/Seo/SeoProductsForArtworks.tsx [hunk 1]
```diff
@@ -14,7 +14,8 @@ const AVAILABILITY = {
   sold: "https://schema.org/OutOfStock",
 }
 
-const formatCurrency = value => currency(value, { separator: "" }).format()
+const formatCurrency = value =>
+  currency(value / 100, { separator: "" }).format()
```
Comment (zephraph) [inline src/Apps/Collect/Components/Seo/SeoProductsForArtworks.tsx]: Is this safe? Specifically for the currencies that aren't divisible by 100. cc @yuki24 cause I know he's currently in the currency working group.

Comment (damassi) [reply src/Apps/Collect/Components/Seo/SeoProductsForArtworks.tsx]: `currency` handles this

Comment (yuki24) [reply src/Apps/Collect/Components/Seo/SeoProductsForArtworks.tsx]: We should probably be more careful about this (or JPY should be normalized to align with other currencies). I'm looking up works with other currencies such as KRW and CNY and will report back.

currencies

Comment (damassi) [reply src/Apps/Collect/Components/Seo/SeoProductsForArtworks.tsx]: Can we merge this PR in the meantime?

Comment (yuki24) [reply src/Apps/Collect/Components/Seo/SeoProductsForArtworks.tsx]: South Korean won is also not divided:

Screen Shot 2019-07-31 at 4 25 10 PM

Comment (yuki24) [reply src/Apps/Collect/Components/Seo/SeoProductsForArtworks.tsx]: Please go ahead and merge.

**Comment by artsyit** [comment]
:rocket: PR was released in v16.20.0 :rocket:

**Pull request merged**